### ISSUE TITLE
[Bug]: can't conda install via environment.yml on mac

### ISSUE THREAD

**Issue by tinosulzer** [pybamm-team/liionpack#130]
### liionpack Version

n/a

### Python Version

3.9

### Describe the bug

Install command suggested in the readme fails
```
$ conda env create -f environment.yml
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound: 
  - redis
 ```

### Expected behaviour

Installs

### Additional context

Mac OS on M1

Possibly related to https://stackoverflow.com/questions/58219956/how-to-fix-resolvepackagenotfound-error-when-creating-conda-environment

**Comment by wigging** [comment]
This may be an issue with installing on the new Apple Silicon M1 Mac. Is Anaconda/Conda supported on M1 Macs? I have an Intel MacBook Pro and I don't have any liionpack installation issues with conda install using the `environment.yml` file.

**Comment by tinosulzer** [comment]
I was able to install without any problems by first creating an empty environment then pip installing into it. Have never tried doing it with an `environment.yml` though, maybe that isn't supported. Also I have to use miniforge, don't know if that makes a difference

**Comment by wigging** [comment]
Did you create the empty environment using conda or pip?

**Comment by tinosulzer** [comment]
conda

**Comment by wigging** [comment]
Can you try to install liionpack using the pip venv approach on your M1 Mac and see if you have any issues?

```
# Clone the repository
$ git clone https://github.com/pybamm-team/liionpack.git

# Create a virtual environment in the repository directory
$ cd liionpack
$ python -m venv .venv

# Activate the virtual environment and upgrade pip if venv installed an old version
$ source .venv/bin/activate
$ pip install --upgrade pip

# Install the required packages
$ pip install -r requirements.txt

# Install the liionpack package from within the repository
$ pip install -e .
```

**Comment by tinosulzer** [comment]
Yeah that works

**Comment by wigging** [comment]
So this looks like a conda issue with M1 Macs and not an issue with liionpack itself. Also, I don't see an Anaconda installer for M1 Macs (https://www.anaconda.com/products/individual). I see a Miniconda installer for the M1 Mac that was released in November for Python 3.8.

**Comment by tinosulzer** [comment]
Ok, thanks for following up
